### PR TITLE
Fix gh-309: Migrate Privacy Policy

### LINKED ISSUE #309: Migrate - Privacy Policy

### PR DESCRIPTION
**Pull request by TheGrits**
Rebased and updated version of #471, to resolve #309.

HTML tested locally on Chrome 50 (ChromeOS)

@rschamp @thisandagain @mewtaylor

### CODE REVIEW

**Comment by TheGrits** [comment]
Wow Travis is so helpful with the lint error... not. Can someone help me figure out what's wrong?

**Comment by rschamp** [comment]
Ah, sorry @TheGrits!  This is my fault, don't worry — Travis will be broken until we merge #476... FYI here are the lint errors:
```
./src/views/privacypolicy/privacypolicy.jsx
    2:5   error  "FormattedMessage" is defined but never used  no-unused-vars
  156:78  error  Trailing spaces not allowed                   no-trailing-spaces
  157:87  error  Trailing spaces not allowed                   no-trailing-spaces
  187:23  error  Trailing spaces not allowed                   no-trailing-spaces
```

If you fix those then I will merge this without Travis since Travis is broken right now.

**Comment by rschamp** [comment]
@TheGrits If you merge develop then that should fix Travis's eslint problem now.

**Comment by TheGrits** [comment]
@rschamp Done! (Merging in `develop` and fixing the lint errors you mentioned)

**Comment by TheGrits** [comment]
@rschamp Whoa, looks like Travis is having some major issues now.

**Comment by rschamp** [comment]
Looks like it was just a temporary network issue on Travis's side. I reran the build and it worked!  Thanks for working on this!

**Comment by rschamp** [comment]
I found one issue with this one when I QA'ed it myself earlier. There is a copy change:

> You can also reset your password through the Account Reset page.
Should be
> You can also reset your password through the **Password Reset** page.

**Comment by TheGrits** [comment]
@rschamp Whoops, sorry about that. Do you want me to open a new PR to fix that?

File: src/views/privacypolicy/privacypolicy.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                                    For technical reasons, if we are required to transfer the
+                                     data on our servers to another location or organization.
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                    <h3>How can I update my personal information?</h3>
+                    <p>
+                        You can update your password, email address, and country through
+                         the <a href="/account/password_change">Account Settings</a> page.
```
Comment (technoboy10) [inline src/views/privacypolicy/privacypolicy.jsx]: This should probably be `accounts/password_change`.

File: src/views/privacypolicy/privacypolicy.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                         You can also reset your password through the
+                          <a href="/account/password_reset">Account Reset</a>
```
Comment (technoboy10) [inline src/views/privacypolicy/privacypolicy.jsx]: This should also be `accounts/password_reset`.

**Comment by technoboy10** [comment]
@TheGrits Other than that issue, everything looks good! :)

QA tests:
- [ ] All links on page go to the intended destination
- [x] All text looks okay
- [x] Can still log in and log out

**Comment by rschamp** [comment]
@technoboy10 @TheGrits  We'll take care of QA issues that come up before a release, so our contributors don't have to work on our schedule. Just pushed the link fixes to the release branch.

**Comment by rschamp** [comment]
Links now work!

**Pull request merged**